### DISCUSSION TITLE
Output: which pressure levels?

### DISCUSSION THREAD

**Discussion by martin-koehler** [ARM-Development/comble-mip#40] [General]
Hi there,
I am setting up my output for the SCM simulations.  The requirement is pressure levels.  ICON has a terrain following height variable.  Do you have a preferred list of pressure levels?
Thanks, Martin

**Comment by fridlind** [comment]
Hi Martin,

We're requesting that SCM output use a fixed pressure coordinate as the vertical dimension as follows: "pressure at mid-level points (native model levels if fixed, otherwise interpolated to fixed grid of choice)". For completeness, we then also request surface pressure and "pressure at layer top points (used for vertical integrations)". 

We hope this manages to strike a balance between providing output at or near the native model resolution and ease of use when analyzing the evolution of many SCM results at once.

Please let us know if you have any further questions.

Many thanks,
Ann

**Reply by martin-koehler** [reply]
Thanks Ann.  I will then use the ICON model levels over the ocean (no orography) and get the mid-levels in pressure levels to interpolate.  Not sure how I will deal with the changing surface pressure though.  Height levels as for the LES would solve that issue.   Of course I will try pressure levels for now.

**Reply by fridlind** [reply]
Martin,

Our SCM assumes a fixed surface pressure, so I didn't anticipate the difficulty you describe. After discussion at GISS, we have now revised the SCM vertical grid requested to be default model layer numbers. We now request the time-varying layer mid-level pressure and layer top edge pressure as shown [here](https://docs.google.com/spreadsheets/d/1Vl8jYGviet7EtXZuQiitrx4NSkV1x27aJAhxxjBb9zI/edit?usp=sharing).

I'll revise the ModelE3 SCM processing script example shortly.

Thank you for noting this issue, and please let us know if you find any further inconsistencies in SCM output requests.

Many thanks,
Ann

**Reply by martin-koehler** [reply]
Thanks Ann, I can work with this much better.
   - Martin
